IdentityCore/src/pop_manager/MSIDDevicePopManager.m

```diff
@@ -84,7 +84,7 @@ - (NSDictionary *)buildPayloadDict:(NSString *)accessToken
     }};
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->